### PR TITLE
TMA-1231: CL functionality

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: docs/command_line.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+To create a project, you will need an Authorization (Project) Token. You can either enter in during the authentication wizard, or pass it as a command line parameter `-t MYPROJECTTOKEN`
+
+Creating a project is as simple as
+
+`gooddata project create`
+
+You can easily use the new project right away and chain multiple calls to the API using the command line tool and `bash` and `jq`
+
+`gooddata api get $(gooddata project create | jq -r '.project.links.projectFeatureFlags') | jq`
```
Comment (panjan) [inline docs/command_line.md]: We should explain what this does.

File: docs/command_line.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Creating a process
+-------
+
+To create a process, you'll need a GoodData Project (Workspace) ID.
+
+There are three ways to deploy a process. The first way is to deploy a script that has been uploaded to your webdav directory by passing the script path. This works for Ruby scripts as well as CloudConnect graphs.
+
+`gooddata -p project_id process create from_path ./path/to/script | jq`
+
+The second way is to use a deploy string, which refers to an object in the GoodData gerrit repository. For example, this is how you would deploy the latest release brick.
```
Comment (panjan) [inline docs/command_line.md]: The name `gerrit repository` exposes implementation details. Use the documented name `GoodData Appstore` instead. Possibly with a link to https://help.gooddata.com/display/doc/Managing+Projects+via+Appstore+Bricks.

Comment (panjan) [reply docs/command_line.md]: Release Brick is not in the public documentation.

File: docs/command_line.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+`gooddata -p project_id process create from_path '${PUBLIC_APPSTORE}:tag/3.5.0:/apps/release_brick'`
```
Comment (panjan) [inline docs/command_line.md]: The CLI help page (` gooddata process create -h`) mentions neither `from_path` nor `as_component `.

**Comment by panjan** [comment]
LGTM. @Teknikov, can you do the language review, please?

**Comment by Teknikov** [comment]
Would suggest the following minor fixes:
 line 28 - complete list of the available command line parameters is available by running
 line 60 - you can either enter it when using the authentication wizard
 line 66 - This line will create a new project and then fetch its featureFlags.
Otherwise, fine by me. :)

**Comment by panjan** [comment]
Thank you @Teknikov. 🌮

**Pull request merged**